.github/workflows/kvm.yml

```
@@ -58,3 +58,25 @@ jobs:
          cd $GITHUB_WORKSPACE
          cd ..
          rm -rf rusty-hermit
+
+  rusty-hermit-integration_tests:
+    name: KVM Tests
+    runs-on: [self-hosted]
+
+    steps:
+    - uses: hecrj/setup-rust-action@v1
+      with:
+         rust-version: nightly
+         components: 'rustfmt, clippy'
+         targets: 'x86_64-unknown-linux-gnu'
+    - uses: actions/checkout@v2.3.4
+      with:
+         submodules: true
+    - name: Check Cargo availability
+      run: cargo --version
+    - name: Check Cargo download
+      run: cargo download --help || true
+    - name: Install cargo-download
+      run: cargo install cargo-download
```

```suggestion
      run: cargo download --help
      continue-on-error: true
    - name: Install cargo-download
      if: ${{ failure() }}
      run: cargo install cargo-download
```

I'm not 100% sure if the `failure()` check works together with `continue-on-error`.

---

Actually this is just a debug check for now. I'll remove that part completely :grin: